src/riak_kv_pb_timeseries.erl

```diff
@@ -140,14 +138,15 @@ process(#tsputreq{table = Table, columns = _Columns, rows = Rows}, State) ->
                     Error = make_rpberrresp(?E_IRREG, to_string({Class, Exception})),
                     {reply, Error, State}
             end;
-        false ->
-            {reply, make_rpberrresp(?E_IRREG, "Invalid data"), State};
+        BadRowIdxs when is_list(BadRowIdxs) ->
+            ValidationBadRowsString = string:join(lists:reverse(BadRowIdxs),", "),
+            EValidationMessage = flat_format("Invalid data found at row index(es) ~s", [ValidationBadRowsString]),
```

Comment:
Should we specify it is base zero in the error message? I was surprised when I tested it but I am used to base one because of erlang.  Changing the base number doesn't fix it because some might be expecting base zero.